---
output:
  rmarkdown::tufte_handout:
    dev: CairoPDF
    keep_tex: true
title: "Summary"
date: "\\vspace{0 mm}"
header-includes:
  \usepackage{graphicx}
  \usepackage{caption}
  \usepackage{fixltx2e}
  \usepackage{rotating}
documentclass: tufte-handout
classoption: a4paper, sfsidenotes, twoside

bibliography: /home/tom/Dropbox/temp/PapersLib-2015-11-04.bibtex
---

```{r setup, include=FALSE, results="hide"}
library(ggplot2)
library(data.table)
library(grid)
theme_slide <- theme_minimal() +
  theme(plot.background = element_rect(fill = "transparent", colour = NA))
knitr::opts_chunk$set(
  dev.args=list(bg = "transparent"),
  echo = FALSE,
  tidy.opts = list(width.cutoff = 32)
  )
knitr::knit_hooks$set(plot = knitr::hook_plot_tex)
source("src/figures.R")
```

# Introduction

## EvoRepRice

* Evolution and adaptation of reproduction in African and Asian rice
* WP1: Cellular and molecular events related to panicle diversity and ovule development
* WP2: Identification of major genomic regions contributing to branching differences (Asian and African domesticated species)
* WP3: Identification of key regulators of panicle development
* WP4: Contribution of S1 genes to post-zygotic reproductive barrier between *O. sativa* and *O. glaberrima*
* WP5: Phenotypic and genetic diversity in panicle branching in Asian and African varieties

## Talk overview

* Aim of this part of the project is to understand how changes in gene expression specify different meristem types during panicle development
* Studied in species with the best reference annotation
* Found patterns of TF gene expression (and other genes) that suggest involvement in apical vs. axillary meristem identity and specification of determinancy in axillary meristems
* Aim to formalise the network that specifies meristem identity so that we can study how the network has changed during domestication of African and Asian varieties

## Diversity of panicle architecture

* Rice panice:
    * Main axis (rachis)
    * A number of primary and secondary branches that carry spikelets
    * Different panicle morphologies between and within species with different branching complexities and grain yields
    
## Yield is related to branching complexity

* Phenotyping of panicles from different accessions of domesticated and wild African and Asian rice
* Correlation between number of secondary branches and number of spikelets
* Number of secondary branches --> complexity
* Number of spikelets --> yield

## Meristem transitions

* Complexity of branching is specified at v. early stages of development
* During this process the activity of different meristem types affects the amount of branching
* Transverse sections of early panicles in *O. sativa* ssp. *japonica* cv. Nipponbare
* N1: Rachis meristem after transition to the reproductive stage (inflorescence meristem)
* N2: Differentiation of primary branch meristems in spiral pattern around main axis
* N3: After formation of all PBs they elongate but the apex retains meristem activity and can produce secondary branches and spikelets
    * At this stage it's not possible to distinguish SBM and SM
* N4: eventually the PBMs and SBMs are converted to SM to form terminal spikelets
    * At this point branching complexity is fixed and elongation and floral organ differentiation occurs
* Han overview
    * Timing of SM specification seems to be controlled and is related to branching complexity (delay in SM specification --> more time for branching) 

## Gene expression specifies meristem activity

* *TAWAWA1*
    * ALOG (Arabidopsis *LSH* and Oryza *G1*): Small family of transcriptional activators
    * In o/e of *TAW1* BM activity is enhanced and SM specification is delayed, resulting in a prolonged branch formation stage and an inflorescence with more branches
    * overexpression has more secondary branches -> *TAWAWA1* suppresses SM identity
* *LOG1*
    * LONELY GUY (LOG) encodes a cytokinin-activating enzyme
    * absence of LOG results in early termination of IM and BMs, which reduces branching complexity

## Gene interactions are complex 

* ABCs in *Arabidopsis*: specification of floral organ identity
    * Original model: A alone specifies sepals, A+B specifies petals, B+C specifies stamens, and C alone specifies carpels
    * Became clear that other genes are involved in these processes, the expression patterns are not clearly defined, and the genes interact to affect each other's expression
* ABCs in rice:
    * ABC model may be applicable to grass flowers
    * Because of complexity and redudancy of interactions it becomes difficult to understand by looking at individual genes or interactions
* A genome-wide approach may be helpful to study meristem transitions
    * Gain a formal understanding of the interactions within larger "regulatory modules"   

# Results

## Laser Microdissection

* Start by describing gene expression in different meristem types
* Technique to capture very small amounts of material for analysis
* Dissected four meristem types: RM, PBM, ePBM and SBM, SM
* Three biological replicates
* Able to harvest 2 - 80 ng of RNA for each replicate
* Amplified RNA for use in sequencing

## Whole genome RNA seq

* mRNA is selected (e.g. by polyA) and reverse-transcribed into cDNA
* cDNA is broken into smaller fragments and adaptors are ligated
* adaptors contain a primer binding site which allows the sequencing instrument to read a short length of the cDNA fragment (50 bp)
* End up with a library of short reads produced from the cDNA fragments
* The number of reads originating from a gene is proportional to the amount of mRNA from that gene in the original sample, i.e. its expression level
* To quantify the number of reads coming from each gene, the library of reads is aligned to the reference genome and the number of reads overlapping each gene is counted

## Quantifying gene expression

* result is a large table containing the number of reads that aligned to each gene in the genome
* the number of reads may be normalised to take into account the differences in length of each gene and the differences between the sequencing depth for each library
* the normalised value can be used to represent the expression of each gene
* a cutoff can be set *e.g.* to determine the number of genes expressed in each sample

## Dynamics of gene expression

* One method is to do a differential expression analysis where a statistical test gives you a *p*-value for each gene
* Hierarchical clustering can be more interesting because it reveals clusters of coexpressed genes
* What are the common patterns of expression in the meristem types

## Coexpression of related genes

* 1 & 4: genes that increase or decrease in expression steadily during development 
* 7 & 5: genes that change expression between the apical meristem (RM) and axillary meristem (PBM–SM) samples
* 6 & 2: genes that change expression gradually over the course of changes in axillary meristem identity
* 3 & 8: weaker clusters containing fewer genes, which had complex expression patterns involving changes in expression particular to the PBM
* co-expression of genes that are involved in inflorescence development along with other uncharacterized TF genes indicates that the clusters capture biologically meaningful groups of genes
* genes involved in hormone signalling and TFs

## Coexpression of TFs

* Clusters also enriched for several TF families

## SPL genes

* *WEALTHY FARMER’S PANICLE* overexpression associated with increased branching and grain yield
* Other SBPs in a pattern that suggests involvement determinate/indeterminate meristem specification

## Expression of TF genes

* Gene set enrichment analysis
* for comparing the **strength** of expression between sets of genes 
* Notable enrichment of SPL in the PBM and depletion in the SM -> indeterminate identity
* several TF families change in expression between apical and axillary meristems
* overall, homeoboxes are more strongly expressed in the SM -> could be involved in SM identity

## Homeobox genes

* pattern of expression of individual homeobox genes, colours relate to clades
* large group of HBs have their strongest expression in SM, no particular clade pattern
* large group of HBs are lowest expressed in the RM (apical), here there seems to be an overrepresentation of Class IV HD-Zip genes

## Class IV HD-ZIPs

* 5 of the 8 expressed Class IV HD-ZIP genes are expressed in this pattern
* Could be involved in identity of apical vs axillary meristems

## Summary

* results are a preliminary step to understanding the control of transistions in meristem identity
* clear switch b/w apical and axillary meristems involving several TF families 
* identified other families that may be involved in the specification of determinate/indeterminate axillary meristem identity
* Also identified dynamically expressed genes related to hormone signalling

## Future work

* Hormone and metabolite measurement in meristems or panicles
* Produce a formal model of how the genes in the "regulatory module" interact to specify meristem types
* One goal is to understand how this network has evolved during different and evolution of African and Asian rice species
    * having a formal model will allow changes in these species to be understood

